src/main/java/dev/walshy/sfmetrics/charts/DisabledItemsChart.java

```diff
@@ -0,0 +1,60 @@
+package dev.walshy.sfmetrics.charts;
+
+import dev.walshy.sfmetrics.SlimefunMetricsChart;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import org.bstats.charts.AdvancedPie;
+import org.bstats.json.JsonObjectBuilder;
+import org.bukkit.Server;
+
+import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This {@link AdvancedPie} shows us what {@link SlimefunItem Items} have been disabled on the
+ * {@link Server}.
+ * <p>
+ * This allows us to see what Items are very unpopular and may be considered for rework/removal.
+ *
+ * @author TheBusyBiscuit
```

Comment:
Hey Biscuit, nice name.

```suggestion
 * @author Walshy
```

Reply:
I did not copy and paste this :eyes: